composer.json

```diff
@@ -101,7 +102,7 @@
         "prestashop/ps_supplierlist": "^1.0",
         "prestashop/ps_themecusto": "^1",
         "prestashop/ps_viewedproduct": "^1.2",
-        "prestashop/ps_wirepayment": "^2",
+        "prestashop/ps_wirepayment": "dev-dev",
```

Comment:
Euh... No !

Reply:
Yeah, we need to release the module, I know 😅

Reply:
@florine2623 is locked : PrestaShop/ps_wirepayment#75